client/js/actions/snippet-actions.js

```diff
@@ -8,14 +8,15 @@ class SnippetActions {
     //   'create'
     // );
   }
+
   getAll () {
-    axios.get('/snippets')
+    return axios.get('/snippets')
```

Comment:
@vrael560 you don't need to return this promise (and definitly when want to have flux arch, you shouldn't)